dao/pom.xml

```diff
@@ -50,6 +50,10 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
```

Comment:
1. Don't think that return JSON from DB is a good idea at all
2. Even if it's JSON. As far as I understood REST endpoint also converts the same classes to JSON again. Maybe it might be simplified?

Reply:
Ok, I'll fix it